resources/views/auth/login.blade.php

```diff
@@ -68,6 +68,94 @@
                 </div>
             </div>
         </div>
+
+        <div class="col-6 col-lg-6 col-md-12 col-sm-12">
```

Comment:
The register code was placed with the login code. This allowed both forms to be on the same page.